src/GridFieldOrderableRows.php

```diff
@@ -138,6 +148,28 @@ public function setImmediateUpdate($bool)
         return $this;
     }
 
+    /**
+     * @see $republishLiveRecords
+     *
+     * @return boolean
+     */
+    public function getRepublishLiveRecords()
+    {
+        return $this->republishLiveRecords;
+    }
+
+    /**
+     * @see $republishLiveRecords
+     *
+     * @param boolean $bool
+     * @return GridFieldOrderableRows $this
```

Comment:
```suggestion
     * @return $this
```

Reply:
This was just for consistency with the method above it